docker/enterprise/Dockerfile

```
@@ -0,0 +1,185 @@
+# -------------------------------------------------------------------------------------------------
+#
+# layer for download and verifying
+FROM debian:buster-slim as graylog-downloader
+
+ARG VCS_REF
```

If we add this, it should get populated in the labels. 

```suggestion
ARG VCS_REF
ARG BUILD_DATE
```